fulp_modules/main_features/bloodsuckers/code/gamemodes/monsterhunter_gamemode.dm

```diff
@@ -23,7 +23,7 @@
 	fakeable = FALSE
 
 /datum/round_event/monster_hunters/start()
-	for(var/mob/living/carbon/human/H in shuffle(GLOB.player_list))
+	for(var/mob/living/H in GLOB.player_list)
```

Comment:
This makes it so simple mobs can become antagonist.

The entire code is kinda wacky as well, you'd want to fix it.